### PR TITLE
simplify nominee election pallet

### PR DESCRIPTION
**Pull request by GopherJ**
companion pr: https://github.com/parallel-finance/nominate-client/pull/6 and https://github.com/parallel-finance/parallel-dapp/pull/342

### CODE REVIEW

File: runtime/heiko/src/lib.rs [hunk 1]
```diff
@@ -958,11 +957,6 @@ type EnsureRootOrHalfCouncil = EnsureOneOf<
     EnsureRoot<AccountId>,
     pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>,
 >;
-type EnsureRootOrHalfTechnical = EnsureOneOf<
```
Comment (kaichaosun) [inline runtime/heiko/src/lib.rs:961]: Still need this origin for other pallets, like market params change.

**Review by kaichaosun — APPROVED** [review]
Well done. 👍

File: runtime/heiko/src/lib.rs (hunk 1 above)
Comment (GopherJ) [reply runtime/heiko/src/lib.rs:961]: @kaichaosun yes, in last commit I'd like to use it for `ScheduleOrigin` but finally found that's in pallet-schedule it seems it hardcoded `ensure_root`, and it's the only place that I used EnsureRootOrHalfTechnical, so I remove it for now

**Comment by GopherJ** [comment]
I removed whiltelist validator set because currently we don't have a clear plan for it

**Pull request merged**